ONSdigital/census-rh-ui
# Cr 1143 add large print option

# Motivation and Context
Adds the ability to request a large print paper form

# What has changed
Addition of check box for large print to the name/address confirmation screen. If ticked, get fulfilment and request fulfilment calls to RHSvc request 'LARGE_PRINT' instead of 'QUESTIONNAIRE'

# How to test?
From /en/requests/paper-form/enter-address/ complete paper form request as before, but tick the new checkbox on the confirm name/address screen
16 new unit tests added

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1143
